server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -217,6 +217,8 @@ public class Manager extends AbstractServer
   private final AtomicBoolean managerInitialized = new AtomicBoolean(false);
   private final AtomicBoolean managerUpgrading = new AtomicBoolean(false);
 
+  private ExecutorService tp = null;
```

Comment:
Should probably give this a better name since its being promoted from a method variable to an instance variable.

Reply:
Addressed in 967df31